src/components/Modals/Onboarding/OnboardingReview.vue

```diff
@@ -54,13 +54,11 @@
           </div>
         </div>
         <div class="onboarding-subHeader2-fillRow">
-          <span class="onboarding-subHeader2-review"> Your Major</span>
+          <span class="onboarding-subHeader2-review"> Your Undergraduate Degree</span>
         </div>
         <div class="onboarding-selectWrapper">
           <div class="onboarding-selectWrapper-review">
-            <label class="onboarding-label"
-              >College<span class="onboarding-required-star">*</span></label
-            >
+            <label class="onboarding-label">College</label>
             <div>
               <label class="onboarding-label--review">{{ collegeText }}</label>
```

Comment:
The placeholder text for College shows if I'm just doing a grad program. It seems for major and minor we use `getMajorFullName` and `getMinorFullName` so I would take a look at those to show an empty string for college. 

![Screen Shot 2021-05-11 at 9 08 15 AM](https://user-images.githubusercontent.com/54298311/117820316-6c4f9680-b238-11eb-882d-acd07703e111.png)